advocacy_docs/edb-postgres-ai/migration-etl/data-migration-service/supported_versions.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: "Product compatibility"
+description: Verify that your Oracle or Postgres database version is compatible with EDB Data Migration Service.
+---
+
+<div id="supported_versions" class="registered_link"></div>
+
+## Supported source databases
+
+The following database versions are supported.
+
+| Database version | Supported |
+|------------------|-----------|
+| Postgres 11 - 16 | Y         |
+| EPAS 11 - 16     | Y         |
+| Oracle 11g       | Y         |
+| Oracle 12c       | Y         |
+| Oracle 18c       | Y         |
+| Oracle 19c       | Y         |
+| Oracle 21c       | Y         |
+
+### Oracle
+
+The EDB Data Migration Service (EDB DMS) stack requires the Oracle database to have archive log mode enabled and supplemental logging data enabled at the table and database level. For details, see [Preparing Oracle source databases](getting_started/preparing_db/preparing_oracle_source_databases).
+
+Container databases (CDB/PDB) and non-CDB sources are supported.
+
+### Postgres/EDB Postgres Advanced Server
+
+Postgres and EDB Postgres Advanced Server sources require a database role or user that can manage replications. For details, see [Preparing Postgres source databases](getting_started/preparing_db/preparing_postgres_source_databases).
+
+## Supported target databases 
```

Comment:
@mlewandowski-edb  I added this section, WDYT? 
@ksnavely @jperozoa @zahid75  could you please confirm the text under this ## section is accurate?

Reply:
Looks fine to me.